GOSS_tests_for_sbps.md

```diff
@@ -0,0 +1,52 @@
+# Assumptions before running this test: 
+
+Ansible scripts are run successfully which configures iSCSI on selected worker nodes and labels them with 'iscsi=sbps'. Goss tests use this label as identification of nodes on which this test has to run. 

```

Comment:
You can't see it, but this whole file has `\r\n` line terminations, they should be just `\n`.  Please strip out all the `\r`s from this file.  One way to do this is:
```
# sed 's/\r//' <GOSS_tests_for_sbps.md >xx
# mv xx GOSS_tests_for_sbps.md 
```

Reply:
done